mlrun/runtimes/sparkjob.py

```diff
@@ -232,7 +232,12 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
                             meta.name, driver, status
                         )
                     )
-                    logger.info("use .watch({}) to see logs".format(meta.name))
+                    resp = self.get_job(meta.name, meta.namespace)
+                    ui_ingress = get_in(resp, "status.driverInfo.webUIIngressAddress")
+                    if ui_ingress:
+                        runobj.status.status_text = "UI is available while the job is running: http://{}".format(
+                            ui_ingress
+                        )
```

Comment:
Use python 3 string formatting
```suggestion
                        runobj.status.status_text = f"UI is available while the job is running: http://{ui_ingress}"
```
Please set the `runobj.status.host` as well so the address will be accessible from the Run object

Reply:
done